app/views/languages/_about.erb

```diff
@@ -5,7 +5,7 @@
   If you've downloaded the <a href='/cli'>command-line client</a> and have <%= track.language %> installed
   on your machine, then go ahead and fetch the first problem.
 </p>
-<%= syntax "exercism #{track.id} #{track.problems.first.slug}", "plain" %>
+<%= syntax "exercism fetch #{track.id}", "plain" %>
```

Comment:
@kotp suggested I remove it, so I did. You might need to "show outdated" above.

Once I'd removed it I had the above thought, but didn't care enough either way to change it again.